Cesium3DTilesSelection/include/Cesium3DTilesSelection/Tileset.h

```diff
@@ -404,13 +408,40 @@ class CESIUM3DTILESSELECTION_API Tileset final {
       const std::vector<ViewState>& frustums,
       float deltaTime = 0.0f);
 
+  /**
+   * @brief Updates a view group, returning the set of tiles to render in this
+   * view. Unlike {@link updateViewGroup}, this method blocks the calling thread
+   * until all tiles suitable for the views have been loaded.
+   *
+   * This method is significantly slower than {@link Tileset::updateView} and
```

Comment:
Refer to the non-deprecated counterpart?

```suggestion
   * This method is significantly slower than {@link updateViewGroup} and
```